### ISSUE TITLE
macOS 10.12 - no audio after install

### ISSUE THREAD

**Issue by Elijen** [toleda/audio_CloverALC#85]
Hi toleda, thanks for all the hard work on this repo!

After updating to Sierra I have no audio, even after running `audio_cloverALC-120.command.zip`. I am out of ideas why. Can you help?

Description:
No audio output (no devices found), input works fine.

System info:
OS: macOS 10.12
GPU: Gigabyte GeForce GTX 970 0 MB
MB: Gigabyte GA-Z97-D3H
CPU: i5-4690

What I did:
Run `audio_cloverALC-120.command.zip` (y to all)
Added kext-dev-mode=1 flag

IOReg:
[IOReg.zip](https://github.com/toleda/audio_CloverALC/files/486804/IOReg.zip)

Clover config:
[config.plist.zip](https://github.com/toleda/audio_CloverALC/files/486808/config.plist.zip)

Boot log:
[bootlog.log.txt](https://github.com/toleda/audio_CloverALC/files/486809/bootlog.log.txt)

DPCI:
![screen shot 2016-09-22 at 10 48 20 am](https://cloud.githubusercontent.com/assets/1304054/18741962/225a49f6-80b2-11e6-90df-728eae9d3005.png)

System info:
![screen shot 2016-09-22 at 10 50 21 am](https://cloud.githubusercontent.com/assets/1304054/18742012/62aaa294-80b2-11e6-8fdd-3527072bec61.png)

**Comment by toleda** [comment]
Attach installed AppleHDA.kext.

**Comment by diepnt90** [comment]
try to plug audio jack into each output audio and choose each output on sound setting.

**Comment by toleda** [comment]
@diepnt90: Start a new Issues thread; see README/Problem Reporting

**Comment by diepnt90** [comment]
@toleda no no, i just also guided this guy to resolve as i resolved it before.

**Comment by toleda** [comment]
His IOReg shows no devices. Nothing to select, not helpful

**Comment by Elijen** [comment]
@toleda I am not sure if I omitted an important step before. Was I supposed to install a custom AppleHDA or AppleALC kext? I thought your patch needed the original AppleHDA.

I managed to make the sound work. Here is what I did:

1) Install AppleALC.kext downloaded from here: https://github.com/vit9696/AppleALC/releases
2) In Clover boot screen I always have to manually set "load kext" and "no cache" (I didn't find out how to do this via config.plist)
